### ISSUE TITLE
HowTo transfer Icons & co. from one device to an other?

### ISSUE THREAD

**Issue by EmEiBee** [Blueforcer/awtrix3#647]
Today I got my second TC001. It's ready to go.
However, I would like to save myself the trouble of entering all the icons and sounds individually into the new TC001.

Hence my question:
Is there a clever way to bring the two pixel clocks up to the same level in this regard?
Or even better:
Is there an option to declare one pixel clock as the master and to bring about an automatic synchronization?

**Comment by Blueforcer** [comment]
Just use the backup function in the Webinterface

**Comment by EmEiBee** [comment]
Uhhh... Ty... Haven't think about that ^^

**Comment by EmEiBee** [comment]
Ok, that works so far but does not set the other things, such as changing the colors of the date display or similar things

**Comment by Blueforcer** [comment]
It just transfers files from the flash. API settings must be executed again.

**Comment by EmEiBee** [comment]
Ok... I just assumed that these settings were stored somewhere and could be copied...
It's a shame, but it can't be changed.

Thank you again for your help

**Comment by EmEiBee** [comment]
... maybe I'm too stupid for that :(
I'm currently trying to put together a code snippet that I can send to a (new) TC001 using "MQTT Explorer" after I've already adjusted most of it using the backup. But somehow it doesn't work.
What's wrong with it?

pixel2/settings
```
{
  "WDCA": "#af7ac5",
  "WDCI": "#797d7f",
  "BRI": 200,
  "ABRI": true,
  "TIM": true,
  "DAT": true,
  "HUM": false,
  "TEMP": false,
  "BAT": false,
  "VOL": 30
}
```
![img-2024-12-19-20-48-15](https://github.com/user-attachments/assets/205870a0-bd74-43ec-865a-bb066938c7a1)

The only thing I've achieved is that the automatic brightness no longer works... Great job ^^
